### PR TITLE
Cross mio updates for Pioneer MAB

### PR DESCRIPTION
**Pull request by rtravis-whoi**
Round 1 of MAB prepwork:

- array_bulk_load - added NES to 'original' Pioneer and added Pioneer MAB array.  We went sequentially with the ASSET_UID & LEGACY_ASSET_UID.  Array_geometry will come later; and we did not update any other arrays (such as with decommission dates for GA, GS, Pioneer NES).  Focus here is getting Pioneer MAB framework in place.

- eng_vocab & node_vocab - addition of Prawler (may be more updates to come at a later date)

- platform_vocab - added NES to 'original Pioneer moorings/mobile assets; and added new Pioneer MAB moorings/mobile assets

working ref des file attached for reference (for new site/site names).  
[Proposed_MAB_RefDes_2024-01-02.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/13889021/Proposed_MAB_RefDes_2024-01-02.xlsx)

Adding a second reference file for sensor_vocab.csv updates (reviewed/confirmed by Sheri)...
[sensor_vocab_additions_2024-01-11.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/13904929/sensor_vocab_additions_2024-01-11.xlsx)

**UPDATED sensor_vocab reference Ss (additions/edits in red font)...

[sensor_vocab_additions_2024-01-11_revised.xlsx](https://github.com/WHOIGit/ooicgsn-asset-management/files/13919432/sensor_vocab_additions_2024-01-11_revised.xlsx)

### CODE REVIEW

File: bulk/array_bulk_load-AssetRecord.csv [hunk 1]
```diff
@@ -1,8 +1,9 @@
...
 N00293,N00293,Array,0,Global Irminger Sea,GI Array Parent,,,,,,,,"POLYGON ((-39.97155 59.65167, -39.97155 60.03320, -39.18295 60.03320, -39.18295 59.65167, -39.97155 59.65167))",20160603,,Woods Hole Oceanographic Institution
 N00294,N00294,Array,0,Global Station Papa,GP Array Parent,,,,,,,,"POLYGON ((-144.94619 49.88512, -144.94619 50.41831, -144.09823 50.41831, -144.09823 49.88512, -144.94619 49.88512))",20160603,,Woods Hole Oceanographic Institution
 N00295,N00295,Array,0,Global Southern Ocean,GS Array Parent,,,,,,,,"POLYGON ((-89.83694 -54.56587, -89.83694 -53.98346, -88.75162 -53.98346, -88.75162 -54.56587, -89.83694 -54.56587))",20160603,,Woods Hole Oceanographic Institution
 N00296,N00296,Array,0,Cabled Array,RSN Array Parent,,,,,,,,"POLYGON ((-130.07880 44.45932, -130.07880 45.98948, -125.08603 45.98948, -125.08603 44.45932, -130.07880 44.45932))",20160603,,University of Washington
+N00297,N00297,Array,0,Coastal Pioneer MAB,CP Array Parent,,,,,,,,,,,Woods Hole Oceanographic Institution
```
Comment (reedan88) [inline bulk/array_bulk_load-AssetRecord.csv:9]: Looks good.

File: bulk/eng_vocab.csv [hunk 2]
```diff
@@ -1,6 +1,7 @@
 DESCRIPTION OF EQUIPMENT,Manufacturer,Model
 Coastal Surface Piercing Profiler Controller,WET Labs,AMP
 Coastal Wire-Following Profiler Controller,McLane,MMP-7
+Coastal Wire-Following Profiler Controller,McLane,PRAWLER
```
Comment (reedan88) [inline bulk/eng_vocab.csv:4]: Looks good

File: bulk/node_vocab.csv [hunk 3]
```diff
@@ -7,6 +7,7 @@ Coastal Glider #,Teledyne Webb,DG-1000
 Open Ocean Glider #,Teledyne Webb,ASSY 4495
 Profiling Glider #,Teledyne Webb,ASSY 302392-3
 Coastal Wire-Following Profiler,McLane,MMP-7
+Coastal Wire-Following Profiler,McLane,PRAWLER
```
Comment (reedan88) [inline bulk/node_vocab.csv:10]: Looks good

File: bulk/platform_vocab.csv [hunk 4]
```diff
@@ -71,5 +71,17 @@ Global Station Papa Apex Profiler Mooring,WHOI,GP02HYPM
...
+Coastal Pioneer MAB Southern Surface Mooring,WHOI,CP11SOSM
+Coastal Pioneer MAB Central Shallow Water Mooring,WHOI,CP12CNSW
+Coastal Pioneer MAB Western Shallow Water Mooring,WHOI,CP12WESW
+Coastal Pioneer MAB Eastern Profiler Mooring,WHOI,CP13EAPM
+Coastal Pioneer MAB Northern Profiler Mooring,WHOI,CP13NOPM
+Coastal Pioneer MAB Southern Profiler Mooring,WHOI,CP13SOPM
+Coastal Pioneer MAB Northeastern Profiler Mooring,WHOI,CP14NEPM
+Coastal Pioneer MAB Southeastern Profiler Mooring,WHOI,CP14SEPM
+Coastal Pioneer MAB Mobile Assets - Gliders,Teledyne Webb,CP15MOAS
+Coastal Pioneer MAB Mobile Assets - AUVs,Hydroid,CP16MOAS
```
Comment (reedan88) [inline bulk/platform_vocab.csv:87]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good. The reference designators and vocab all match the Proposed MAB RefDes sheet as far as I can tell.

**Comment by rtravis-whoi** [comment]
[Pioneer MAB Array AM [DT-227]](https://trello.com/c/Aqg3aPfT/227-pioneer-mab-array-am-dt-227)

File: bulk/sensor_vocab.csv [hunk 5]
```diff
@@ -102,14 +105,20 @@ Photosynthetically Available Radiation: PARAD Series J,WET Labs,ECO PARS
...
+Seafloor Pressure: PRESF Series D,RBR,quartz3QPlus (P-55)
+Seafloor Pressure: PRESF Series E,RBR,quartz3QPlus (P-125)
+Seafloor Pressure: PRESF Series F,RBR,quartz3QPlus (P-1350)
 Seawater pH: PHSEN Series A,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series D,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series E,Sunburst Sensors,SAMI-pH
 Seawater pH: PHSEN Series F,Sunburst Sensors,SAMI-pH
+Suspended Particulates: PRTSZ Series A,Sequoia,LISST-200X
 Short-Period Ocean Bottom Seismometer 1 to 100Hz: OBSSP Series A,Guralp,CMG-6TF
+Turbidity: TURBD Series A,Sea-Bird Electronics,ECO
```
Comment (reedan88) [inline bulk/sensor_vocab.csv:121]: Looks good.

**Review by reedan88 — APPROVED** [review]
Looks good!

File: bulk/platform_vocab.csv (hunk 4 above)
Comment (snwhite) [reply bulk/platform_vocab.csv:87]: The additions look good, but it looks like they should be above as this list looks like it is supposed to be alphabetical.  So I think the MAB stuff should go in front of NES.

File: bulk/array_bulk_load-AssetRecord.csv (hunk 1 above)
Comment (rtravis-whoi) [reply bulk/array_bulk_load-AssetRecord.csv:9]: we can add that info later (my opinion)

File: bulk/platform_vocab.csv (hunk 4 above)
Comment (rtravis-whoi) [reply bulk/platform_vocab.csv:87]: ARgggggghhh ok - rearranged; plus had to rearrange some NES ones that were not alphabetical...

File: bulk/sensor_vocab.csv [hunk 6]
```diff
@@ -31,6 +31,8 @@ ADCP Velocity Profiler (short range): ADCPT Series E,Teledyne RDI,WHM150-6000
 ADCP Velocity Profiler (short range): ADCPT Series F,Teledyne RDI,WHS150-1500
 ADCP Velocity Profiler (short range): ADCPT Series G,Teledyne RDI,WHS150-1500-I
 ADCP Velocity Profiler (short range): ADCPT Series M,Teledyne RDI,WHS600-500-Waves
+ADCP Velocity Profiler (short range): ADCPT Series Q,Teledyne RDI,WHS600-200 [WHII]
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply bulk/sensor_vocab.csv]: done

File: bulk/sensor_vocab.csv [hunk 7]
```diff
@@ -54,6 +56,7 @@ CTD Mooring (Inductive): CTDMO Series G,Sea-Bird Electronics,SBE 37-IM
 CTD Mooring (Inductive): CTDMO Series H,Sea-Bird Electronics,SBE 37-IM
 CTD Mooring (Inductive): CTDMO Series Q,Sea-Bird Electronics,SBE 37-IM
 CTD Mooring (Inductive): CTDMO Series R,Sea-Bird Electronics,SBE 37-IM
+CTD Mooring (Inductive): CTDMO Series S,Sea-Bird Electronics,SBE 37IM
```
[the remark this answers is not in the document]
Comment (rtravis-whoi) [reply bulk/sensor_vocab.csv]: done

**Review by reedan88 — APPROVED** [review]
Updates look good.

**Review by snwhite — APPROVED** [review]
Updated changes to sensor_vocab.csv look good.

**Pull request merged**